### ISSUE TITLE
[Feat]: Accessibility suggestion for "served fresh..." section on Homepage

### ISSUE THREAD

**Issue by freaksauce** [syntaxfm/website#1381]
The "served fresh thrice weekly" section is currently easy to skip over with a screen reader as they are not focusable elements. I wonder if you were to add a tabindex to the container and an aria-label which summarised the visual information it might be more usable for screen readers?

```
<div tabindex="0" aria-label="Monday, 15 minute Hasty Treat, quick and information episodes">
  <p class="tag length svelte-ftzc5r">15m</p> <p class="tag day svelte-ftzc5r">Monday</p>
  <div>
    <p class="tag svelte-ftzc5r">HASTY TREAT</p>
  </div>
  <p class="desc svelte-ftzc5r">Quick n' Informative</p>
</div>
```

**Comment by stolinski** [comment]
We'll consider re-opening, if we decide this needs a tabindex, but since it isn't interactive, I'm not sure we'd want to do that.

**Comment by freaksauce** [comment]
No worries Scott, hope you don't mind my spamming of a11y tickets I know some of these are probably not a priority at the moment.
